src/components/dex/Pair.vue

```diff
@@ -47,32 +47,17 @@ export default {
         this.$services.neo.getHolding(this.$store.state.currentMarket.baseAssetId).unitValue : 0;
     },
 
-    change24Hour() {
-      return new BigNumber(String(this.close24Hour))
-        .minus(new BigNumber(String(this.tickerData.open24hr)));
-    },
-
-    close24Hour() {
-      const tradeHistory = this.$store.state.tradeHistory;
-      const marketName = this.$store.state.currentMarket.marketName;
-      return tradeHistory[marketName] &&
-        tradeHistory[marketName].trades &&
-        tradeHistory[marketName].trades.length ?
-        tradeHistory[marketName].trades[0].price : 0;
-    },
-
-    percentChangeAbsolute() {
-      return Math.round(((this.change24Hour)
-        / this.tickerData.open24hr) * 10000) / 100;
-    },
-
     tableData() {
       return this.filteredMarkets().map(({ quoteCurrency, marketName }) => {
         const tradeHistory = this.$store.state.tradeHistory[marketName];
         const hasTradeHistory = tradeHistory && tradeHistory.trades && tradeHistory.trades.length > 0;
-        const price = this.$formatTokenAmount(hasTradeHistory ? tradeHistory.close24Hour : 0);
-        const vol = this.$formatNumber(hasTradeHistory ? tradeHistory.volume24Hour : 0);
-        const change = this.$formatNumber(this.percentChangeAbsolute);
+        const close24Hour = this.marketData[marketName].close24Hour || 0;
```

Comment:
is it possible for this.marketData[marketName] to be undefined here?

Reply:
if this were undefined should it be using `this.getClose24Hour(marketName)`

Reply:
Where is this value actually used within the scope it is defined also? It seems like below it just uses `tradeHistory.close24Hour` rather than this const.

Reply:
This looks to still not be addressed.